src/Components/Web/src/UIErrorEventArgs.cs

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNetCore.Components
+{
+    /// <summary>
+    /// Supplies information about an error event that is being raised.
+    /// </summary>
+    public class UIErrorEventArgs : UIEventArgs
```

Comment:
I'm not sure whether this is actually usable. The JS `error` event fires on `window`, but Blazor only lets you register .NET event handlers on DOM elements.

Reply:
Actually forget this individual comment. It's just one example of a broader area we need to make a decision about - commented about that separately.